src/osgOcean/OceanScene.cpp

```diff
@@ -21,6 +21,10 @@
 
 #include <osg/Depth>
 
+
+#define SCREEN_WIDTH 1920

```

Comment:
Let's leave the principle aside for now.

The most pressing question for me would be: what's the performance / quality impact of this change when one does *not* need such a big image ? Our machines are already breaking down when doing simulation ....